Sources/AppStoreConnectCLI/Arguments/BuildLocalizationInputArguments.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+
+struct BuildLocalizationInputArguments: ParsableArguments {
+    @Argument(help: "The locale information of the build localization resource. eg. (en-AU)")
+    var locale: String
+
+    @Option(
```

Comment:
When using this command I don't think you would pass the "whatsNew" content on the command line.  You would either pass it via a file or as standard input (ie, via a pipe).

Reply:
> When using this command I don't think you would pass the "whatsNew" content on the command line. You would either pass it via a file or as standard input (ie, via a pipe).

Hi @orj The command provides two options, you can either input via command line or file path. It might be useful if the input text is short. So do you want me to remove `--whatsNew` option?

Reply:
@DechengMa I believe the suggestion is to remove the `--whatsNew` argument and if there is no file specified to read from `stdin`

Reply:
For a very short (one line) what's new you might have a command line option for it (eg, git has `-m "Commit message"` for commits.  I guess we could keep it for those cases.  But the "unix philosophy" would be to take the input from standard input usually:
```
asc testflight builds localization create com.bundleid 1.0 100 en_AU < changelog.txt
```
You could also do one-liners with:
```
echo "Bugfixes and performance improvements" | asc testflight builds localization create com.bundleid 1.0 100 en_AU
```

Reply:
This was implemented. Now these options are provided 

```
--whats-new "Some short text"
```

Or 
```
 --path ../whatsNew.text
```

Or `stdin`
```
echo "How about test this" | swift run asc testflight builds localization update com.exmaple.foo 1.0.9 5 en-AU   
Reading text from stdin. (press CTRL+D to finish)
+---------+------------------------+
| Locale  | What's New (truncated) |
+---------+------------------------+
|  en-AU  | "How about test this"  |
+---------+------------------------+
```
```
swift run asc testflight builds localization update com.exmaple.foo 1.0.9 5 en-AU                        
[3/3] Linking asc
Reading text from stdin. (press CTRL+D to finish)
Hello  
Please test these 
Cheers 
+---------+-----------------------------------+
| Locale  | What's New (truncated)            |
+---------+-----------------------------------+
| en-AU   | "Hello Please test these  Cheers" |
+---------+-----------------------------------+
```